trackintel/preprocessing/triplegs.py

```diff
@@ -229,6 +232,10 @@ def _generate_trips_user(df, gap_threshold):
         if in_trip is True:
             # during trip generation/recording
 
+            # if the origin activity is unknown, fill origin coordinates with coordinates of first tripleg
+            if origin_activity["geom"] is None:
```

Comment:
Oh shit... this will be horrible to merge with our updated trip generation [PR220](https://github.com/mie-lab/trackintel/pull/220)
The best thing to do might be to wait until we merged it... Unfortunatly it is somewhat stuck with a bug we are working on...

Reply:
Yes let's discuss this on Wednesday